README.md

```diff
@@ -22,14 +33,10 @@ This application has been build by the following developers:
 - Ludovic Dubost, XWiki SAS
 - Maxim Ionut, XWiki SAS
 - Anca Luca, XWiki SAS
+- Alexandru Cotiugă, XWiki SAS
```

Comment:
Yes, this information is useless on GitHub since you can easily view the people that committed on the repository.

The pom.xml and the e.x.o page are the only places where this information would be relevant.